orchestration/dagster_orchestration/hca_manage/diff_dirs.py

```diff
@@ -7,11 +7,13 @@
 import google.auth
 from google.cloud import storage
 
+from hca_orchestration.contrib import google as hca_google
```

Comment:
minor gripe: i find that `import as` often makes it harder to intuit where code is coming from, since it often leads to using inconsistent aliases for a given package across a code base, and i usually learn where code is sourced from by spotting patterns

Reply:
What would an alternative approach be? I often go the route of something like
`from hca_orchestration import contrib` and then use `contrib.google` 🤷🏾